bot.py

```diff
@@ -45,11 +45,27 @@ def build_response_text(rate, bank):
     )
         
 def build_response_image(text):
-    photo = Image.new("I", (600, 670), "white")
-    font = ImageFont.truetype("arial.ttf", 25)
-    drawer = ImageDraw.Draw(photo)
-    drawer.text((10, 10), text, font=font, fill='black')
-    return photo 
+    
+    width = 10000
+    height = 20000
+    starting_position = (10, 10)
+    
+    font_size = 30
+    font = ImageFont.truetype('arial.ttf', font_size)
+
+    image = Image.new('RGB', (width, height), (255, 255, 255))
+    draw = ImageDraw.Draw(image)
+
+    left, top, right, bottom = draw.multiline_textbbox(starting_position, text=text, font=font)
+
+    width = right - left + starting_position[0] * 2
+    height = bottom - top + starting_position[1] * 2
+       
+    image = Image.new('RGB', (width, height), (255, 255, 255))
```

Comment:
Use named constant or color string for defining white/black color

Reply:
Done